pkg/microservice/aslan/core/common/service/fs/s3.go

```diff
@@ -194,3 +194,27 @@ func DownloadAndCopyFilesFromGerrit(name, localBase string, logger *zap.SugaredL
 	}
 	return nil
 }
+
+func DownloadAndCopyFilesFromGitee(name, localBase string, logger *zap.SugaredLogger) error {
+	chartTemplate, err := mongodb.NewChartColl().Get(name)
+	base := path.Join(config.S3StoragePath(), chartTemplate.Repo)
+	if err := os.RemoveAll(base); err != nil {
+		logger.Errorf("Failed to remove dir, err:%s", err)
```

Comment:
If it's not necessary to return erorr, use `logger.Warnf()` instead of `logger.Errorf()`